src/scheduler.rs

```diff
@@ -141,6 +141,16 @@ async fn inbox_loop(ctx: Context, started: Sender<()>, inbox_handlers: ImapConne
                     match ctx.get_config_bool(Config::FetchedExistingMsgs).await {
                         Ok(fetched_existing_msgs) => {
                             if !fetched_existing_msgs {
+                                // Consider it done even if we fail.
+                                //
+                                // This operation is not critical enough to retry,
```

Comment:
I'd say it is up to the user. Can we report in the case of fetch failure some persistent error? Maybe create a device message?

Reply:
Maybe, but i'd personally prefer to see all potentially important warnings. But also i usually don't read logs until i need to debug smth. We can note that if it's the first setup of the app, there's nothing very important to fetch. But if it's not the first setup, shouldn't the user do an export/import instead?

Reply:
@iequidoo I think your suggestion on how we report errors to the user adequately should be discussed in a brighter context. In my opinion, we should never display (annoying) pop-ups to the user, especially not when they are multiple. We could - in a separate PR - introduce e.g. a small yellow triangle with an exclamation mark that is layed on the top left positioned avatar of our own, so the interested user might tap on it to open the "switch email accoiunt" window where we could display the permanent (not resolved) error, perhaps a link to open the log file to view details, too. For now, let's just postpone the decision about the way of error reporting. At least, we currently log any error in the log file.